lib/store.go

```diff
@@ -2,37 +2,73 @@ package lib
 
 import (
 	"sync"
+	"time"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
+var (
+	cleanupLoopInterval = 5 * time.Minute
+	stateExpiry         = 7 * time.Minute // a bit more than an epoch with 6.4 min
```

Comment:
Have we thought of using hard-coded configs rather than a magic number?  If any seconds per slot or slots per epoch change, this will be hard to find

```go
stateExpiry = slotsPerEpoch * secondsPerSlot * time.Second + someBuffer
```

Reply:
updated